tests/PSRule.Tests/PSRule.Baseline.Tests.ps1

```diff
@@ -29,7 +29,18 @@ BeforeAll {
 
     #region TestCases
 
-    $allYamlBaselines = @"
+    $allYamlBaselines = Get-Content -Path (Join-Path -Path $here -ChildPath 'Output.Baseline.yaml') -Raw;
+    $allFourSpaceJsonBaslines = Get-Content -Path (Join-Path -Path $here -ChildPath 'Output.Baseline.4.jsonc') -Raw;
+    $allTwoSpaceJsonBaslines = Get-Content -Path (Join-Path -Path $here -ChildPath 'Output.Baseline.2.jsonc') -Raw;
+    $allZeroSpaceJsonBaslines = Get-Content -Path (Join-Path -Path $here -ChildPath 'Output.Baseline.0.jsonc') -Raw;
```

Comment:
This is great 🚀 . I wanted to do this before and never got around to it for some reason 😄.